superset/connectors/sqla/views.py

```diff
@@ -18,19 +18,19 @@
 """Views used by the SqlAlchemy connector"""
 import logging
 
-from flask import flash, Markup, redirect
+from flask import flash, Markup, redirect,request, Response
 from flask_appbuilder import CompactCRUDMixin, expose
 from flask_appbuilder.actions import action
 from flask_appbuilder.models.sqla.interface import SQLAInterface
 from flask_appbuilder.security.decorators import has_access
 from flask_babel import gettext as __
 from flask_babel import lazy_gettext as _
-
+import simplejson as json
```

Comment:
Seems like a redundant import, please remove it

Reply:
used in json_success(json.dumps({'table_name': table_model.name}))